### PR TITLE
Removed references to Lidgren

### PR DESCRIPTION
**Pull request by AntonyHanna**
Removes references to Lidgren, thats all :)

### CODE REVIEW

File: NitroxServer/ServerAutoFacRegistrar.cs [hunk 1]
```diff
@@ -40,21 +39,18 @@ private static void RegisterCoreDependencies(ContainerBuilder containerBuilder)
             containerBuilder.RegisterType<ConsoleCommandProcessor>().SingleInstance();
 
             containerBuilder.RegisterType<LiteNetLibServer>().SingleInstance();
-            containerBuilder.RegisterType<LidgrenServer>().SingleInstance();
             
             containerBuilder.Register<Communication.NetworkingLayer.NitroxServer>(ctx =>
             {
                 ServerConfig config = ctx.Resolve<ServerConfig>();
```
Comment (Measurity) [inline NitroxServer/ServerAutoFacRegistrar.cs]: Remove the code in this code block and resolve to LitenetLibServer

```suggestion
       return ctx.Resolve<LiteNetLibServer>();
```

**Comment by Measurity** [comment]
Fixes #889

File: NitroxServer/Serialization/ServerConfig.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public class ServerConfig
         private readonly ServerConfigItem<string> serverPasswordSetting = new ServerConfigItem<string>("ServerPassword", "");
         private readonly ServerConfigItem<string> adminPasswordSetting = new ServerConfigItem<string>("AdminPassword", GenerateRandomString(12, false));
         private readonly ServerConfigItem<string> gameModeSetting    = new ServerConfigItem<string>("GameMode", "Survival");
-        private readonly ServerConfigItem<string> networkingType = new ServerConfigItem<string>("NetworkingType", "lidgren");
+        private readonly ServerConfigItem<string> networkingType = new ServerConfigItem<string>("NetworkingType", "litenetlib");
```
[the remark this answers is not in the document]
Comment (AntonyHanna) [reply NitroxServer/Serialization/ServerConfig.cs]: Removed in the latest commit

File: NitroxServer/ServerAutoFacRegistrar.cs (hunk 1 above)
Comment (AntonyHanna) [reply NitroxServer/ServerAutoFacRegistrar.cs]: Also resolved in the latest commit

**Comment by AntonyHanna** [comment]
I believe the latest commit should sort out the two suggestions from your comments, if there's anything else I should change let me know :)

**Pull request merged**